modules/010-prometheus-crd/crds/customalertmanager.yaml

```diff
@@ -99,34 +100,30 @@ spec:
                           default: false
                 internal:
                   type: object
+                  description: Internal Alertmanager parameters.
                   properties:
                     inhibitRules:
-                      description: List of inhibition rules. The rules will only apply to
-                        alerts matching the resource’s namespace.
+                      description: List of inhibition rules.
                       items:
-                        description: InhibitRule defines an inhibition rule that allows
-                          to mute alerts when other alerts are already firing. See https://prometheus.io/docs/alerting/latest/configuration/#inhibit_rule
+                        description: |
+                          InhibitRule defines an inhibition rule that allows to mute alerts when other alerts are already firing ([details...](https://prometheus.io/docs/alerting/latest/configuration/#inhibit_rule)).
                         properties:
                           equal:
-                            description: Labels that must have an equal value in the source
-                              and target alert for the inhibition to take effect.
+                            description: Labels that must have an equal value in the source and target alert for the inhibition to take effect.
                             items:
                               type: string
                             type: array
                           sourceMatch:
-                            description: Matchers for which one or more alerts have to exist
-                              for the inhibition to take effect. The operator enforces that
-                              the alert matches the resource’s namespace.
```

Comment:
Why we remove "The operator enforces that the alert matches the resource’s namespace." ?

Reply:
Fixed